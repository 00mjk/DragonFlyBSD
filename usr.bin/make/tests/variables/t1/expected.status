$DragonFly: src/usr.bin/make/tests/variables/t1/expected.status,v 1.1 2005/02/25 08:53:14 okumoto Exp $
0
