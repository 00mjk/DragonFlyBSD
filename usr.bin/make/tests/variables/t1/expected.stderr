$DragonFly: src/usr.bin/make/tests/variables/t1/expected.stderr,v 1.1 2005/02/25 08:53:14 okumoto Exp $
